docs/jwst/outlier_detection/outlier_detection_imaging.rst

```diff
@@ -1,180 +1,127 @@
+============
 
 This module serves as the interface for applying ``outlier_detection`` to direct
-image observations, like those taken with MIRI, NIRCam and NIRISS.  The code implements the
-basic outlier detection algorithm used with HST data, as adapted to JWST.
-
-Specifically, this routine performs the following operations:
-
-#. Extract parameter settings from input model and merge them with any user-provided values.
-   See :ref:`outlier detection arguments <outlier_detection_step_args>` for the full list
-   of parameters.
-
-#. Convert input data, as needed, to make sure it is in a format that can be processed.
-
-   * A :py:class:`~jwst.datamodels.ModelLibrary` serves as the basic format for
-     all processing performed by
-     this step, as each entry will be treated as an element of a stack of images
-     to be processed to identify bad-pixels/cosmic-rays and other artifacts.
-   * If the input data is a :py:class:`~jwst.datamodels.CubeModel`, convert it into a ModelLibrary.
-     This allows each plane of the cube to be treated as a separate 2D image
-     for resampling (if done) and for combining into a median image.
-
-#. By default, resample all input images.
-
-   * The resampling step starts by computing an output WCS that is large enoug
-     to encompass all the input images.
-   * All images from the *same exposure* will get resampled onto this output
-     WCS to create a mosaic of all the chips for that exposure.  This product
-     is referred to as a "grouped mosaic" since it groups all the chips from
-     the same exposure into a single image.
-   * Each dither position will result in a separate grouped mosaic, so only
-     a single exposure ever contributes to each pixel in these mosaics.
-   * An explanation of how all NIRCam multiple detector group mosaics are
+image observations, like those taken with MIRI, NIRCam, and NIRISS.
+A :ref:`Stage 3 association <asn-level3-techspecs>`,
+which is loaded into a :py:class:`~jwst.datamodels.ModelLibrary` object,
+serves as the basic format for all processing performed by this step.
+This routine performs the following operations:
+
+#. Extracts parameter settings from input models and merges them with any user-provided values.
+   See :ref:`outlier detection arguments <outlier_detection_step_args>` for the full list of parameters.
+
+#. By default, resample all input images to the same output WCS. The resample process is
+   controlled by the ``fillval``, ``pixfrac``, ``kernel``, and ``good_bits`` parameters;
+   see the :ref:`outlier detection arguments <outlier_detection_step_args>` for more information.
+   Resampling can be turned off with the ``resample_data`` parameter.
+
+   * Compute an output WCS that is large enough to encompass all the input images.
+   * Resample all images from the *same exposure* onto this output WCS to create a mosaic of all the chips
+     for that exposure.  This product is referred to as a "grouped mosaic" since it groups all the chips
```

Comment:
```suggestion
   * Resample all images from the *same exposure* onto this output WCS to create a mosaic of all the detectors
     for that exposure.  This product is referred to as a "grouped mosaic" since it groups all the detectors
```